src/properties.jl

```diff
@@ -875,3 +875,53 @@ const UTF8_ATTRIBUTE_PROPERTIES = AttributeCreateProperties()
 
 _link_properties(::AbstractString) = UTF8_LINK_PROPERTIES
 _attr_properties(::AbstractString) = UTF8_ATTRIBUTE_PROPERTIES
+
+struct HDF5Context
```

Comment:
We should move the context stuff to it's own file, `context.jl`.

Reply:
Moved.